vagrant/provisioning/roles/arkcase-prerequisites/tasks/main.yml

```diff
@@ -1,3 +1,13 @@
+- name: set Java 8 as Java Home (old versions versions)
```

Comment:
Consider putting the specific version in the title: `set Java 8 as Java home (versions below 2023.02)`